assets/js/modules/analytics/constants.js

```diff
@@ -18,5 +18,12 @@
 
 export const AREA_MODULE_ANALYTICS_MAIN = 'moduleAnalyticsMain';
 
-// Date that Universal Analytics will stop recording data: July 1, 2023.
+/**
+ * Date that Universal Analytics will stop recording data: July 1, 2023.
+ *
+ * Used to display a line on charts indicating why UA-sourced data has no
+ * data after this date.
```

Comment:
Reading this comment suggests the constant is _only_ used for the charts, could you amend it to make it clear it's used elsewhere too?